.teamcity/settings.kts

```diff
@@ -189,11 +185,6 @@ object ChocolateyGUIQA : BuildType({
             }
         }
 
-        step {
-            name = "Include Signing Keys"
-            type = "PrepareSigningEnvironment"
-        }
-
         script {
             name = "Call Cake"
             scriptContent = """
```

Comment:
In the line below this comment, can you change `build.official.bat` to `build.bat`?  Since the "Include Signing Keys" step has been removed, we shouldn't call `build.official.bat` anymore.